.ruby-style.yml

```diff
@@ -242,6 +245,9 @@ Style/WhenThen:
   Description: Use when x then ... for one-line cases.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#one-line-cases
   Enabled: false
+Layout/EmptyLineAfterMagicComment:
+  Description: Checks for a newline after the final magic comment.
+  Enabled: false
```

Comment:
We can enable it, because that rule can autofix with the `-a` option

Reply:
Perfect 🙂